web/samples/attachment-api/utilities.js

```diff
@@ -74,11 +76,14 @@ function loadKeyboards()
 		detachBtn.value   = 'Detach';
 		div.appendChild(detachBtn);
 		
+		// We need some extra space to properly detect iframe attachment.
+		var attachTimer = elem.tagName.toLowerCase() == "iframe" ? 50 : 1;
```

Comment:
Hmm.  I probably could condition it on an onload event for the iframe from the test page instead.  But yeah, less than that would leave the attachment state inaccurate.  I suppose it could be set to poll until attached if it expected attachment?